Arithmetic/SumAverageRunningInt/Program.cs

```diff
@@ -11,10 +11,11 @@ static void Main(string[] args)
             const int lowerBound = 1;
             const int upperBound = 100;
 
-            for (var number = lowerBound; number <= upperBound; ++number) 
-            {
+            for (var number = lowerBound; number <= upperBound; ++number) {
                 sum += number;
             }
```

Comment:
Vari pārskatīt tukšās rindas.  Pēc vajag ielikt }, bet starp šādiem var nelikt 
decimal height = Convert.ToInt32(Console.ReadLine());

Console.WriteLine("The triangle's area is " + (ground * height) / 2);